commitlint.config.js

```diff
@@ -0,0 +1,116 @@
+const {
+  utils: { getPackages }
+} = require('@commitlint/config-lerna-scopes')
+
+module.exports = {
+  extends: ['@commitlint/config-conventional', '@commitlint/config-lerna-scopes'],
+  rules: {
+    // see: https://github.com/conventional-changelog/commitlint/blob/master/docs/reference-rules.md
+    'scope-empty': [2, 'never'],
+    'scope-enum': async (ctx) => [
+      2,
+      'always',
+      [...(await getPackages(ctx)), 'release', 'deps', 'ci']
+    ]
+  },
+  helpUrl: 'https://github.com/Kong/public-shared-actions',
+  prompt: {
```

Comment:
This is optional, users can run the standard `git commit` command to commit their changes. 

If someone wants to follow the interactive prompt way, they need to run `pnpm commit` instead of `git commit`. I have updated the README with this.